ui/app/AppLayouts/Communities/popups/CreateChannelPopup.qml

```diff
@@ -70,293 +178,456 @@ StatusDialog {
         } else {
             nameInput.input.asset.isLetterIdenticon = true;
         }
-    }
-
-    Connections {
-        enabled: root.opened && root.emojiPopupOpened
-        target: emojiPopup
-
-        function onEmojiSelected(emojiText: string, atCursor: bool) {
-            nameInput.input.asset.isLetterIdenticon = false;
-            nameInput.input.asset.emoji = emojiText
-        }
-        function onClosed() {
-            root.emojiPopupOpened = false
-        }
-    }
 
-    function isFormValid() {
-        return (nameInput.valid &&
-                descriptionTextArea.valid) &&
-                Utils.validateAndReturnError(colorDialog.color.toString().toUpperCase(),
-                                             communityColorValidator) === ""
+        updateRightButtons()
     }
 
-    StatusScrollView {
-        id: scrollView
-
-        property ScrollBar vScrollBar: ScrollBar.vertical
-
-        anchors.fill: parent
-        contentWidth: availableWidth
-        padding: 16
-
-        function scrollBackUp() {
-            vScrollBar.setPosition(0)
-        }
+    width: 640
 
+    readonly property list<Item> discordPages: [
         ColumnLayout {
-            id: content
-
-            width: scrollView.availableWidth
-            spacing: 0
+            id: fileListView
+            objectName: "discordFileListView" // !!! DON'T CHANGE, clearFilesButton depends on this
```

Comment:
what about sth like that:
```suggestion
            readonly property bool isFileListView: true
```
?
Then there is no risk of interfering with tests relying on objectName